src/index.js

```diff
@@ -40,9 +40,10 @@ export default class ReactTextMoreLess extends Component {
     this.setDOM();
   }
 
-  componentWillUpdate() {
+  getSnapshotBeforeUpdate() {
```

Comment:
Is this a proper usage of the `getSnapshotBeforeUpdate` API?

Reply:
yes, take a look at these references:
https://reactjs.org/docs/react-component.html#getsnapshotbeforeupdate
https://stackoverflow.com/a/53194098/6528963